plugins/scaffolder-actions/scaffolder-backend-module-http-request/src/actions/run/helpers.ts

```diff
@@ -78,7 +78,7 @@ export const http = async (
         res.status
       } Response body: ${JSON.stringify(body)}`,
     );
-    throw new HttpError('Unable to complete request');
+    return { code: res.status, headers: {}, body };
```

Comment:
I wonder would any code already depend on the current behaviour?

Maybe it makes sense to add a new option to the action step like the axios `validateStatus: false`.

Otherwise, we should consider releasing this as a major update.

I prefer the validateStatus option.

Reply:
Yes good ideas! I'll add the option